Dockerfile

```diff
@@ -45,7 +47,7 @@ RUN cd caddy/frankenphp && \
 
 ENTRYPOINT ["/bin/bash","-c"]
 
-FROM php:8.2.0RC6-zts-bullseye AS final
+FROM php:${php_base_tag}-bullseye AS final
```

Comment:
Instead of using build-args, we should perhaps move the Dockerfiles into a dedicated directory `/dockerfiles/8.2/Dockerfile`, etc. 

If we need to go back to manually building PHP again for some reason, it would be nice if that didn't affect _every single PHP version_ and it may be further complicated if the build steps are different between PHP versions. FWIW, it's likely to be somewhat different for PHP 8.3 due to some changes in Fibers (new build switches), at least for x86.

I think this would also simplify the matrix somewhat.

Reply:
Maybe should we use the same templating system as official images?

Reply:
https://github.com/docker-library/bashbrew